index.js

```diff
@@ -205,12 +205,12 @@ class Analytics {
 
     if (!this.flushed) {
       this.flushed = true
-      this.flush()
+      this.flush(callback)
       return
     }
 
     if (this.queue.length >= this.flushAt) {
-      this.flush()
+      this.flush(callback)
     }
 
     if (this.flushInterval && !this.timer) {
```

Comment:
There's also another one here:
https://github.com/segmentio/analytics-node/blob/d2c565d27110b8c3846372a8d8aacc69b0ef83ca/index.js#L217

Reply:
thanks, @pbassut
just applied the suggestion!